lexicon/providers/aliyun.py

```diff
@@ -0,0 +1,217 @@
+"""Module provider for Aliyun"""
+from __future__ import absolute_import
+import logging
+import time
+import datetime
+import random
+import hmac
+import sys
+from hashlib import sha1
+from six.moves import urllib
+
+import requests
+from lexicon.providers.base import Provider as BaseProvider
+
+
+LOGGER = logging.getLogger(__name__)
+
+NAMESERVER_DOMAINS = ['hichina.com']
+
+ALIYUN_DNS_API_ENDPOINT = 'https://alidns.aliyuncs.com'
+ALIYUN_DNS_DEFAULT_REGION = 'cn-hangzhou'
+
+def provider_parser(subparser):
+    """Module provider for Aliyun"""
+    subparser.add_argument(
+        "--access-key-id", help="specify api key for authentication")
```

Comment:
Usually we use the convention `--auth-param` in Lexicon for params that contains sensible data for the provider authentication. Could `--access-key-id` be replaced by `--auth-key-id` ?